packages/Panel/src/helpers.js

```diff
@@ -53,3 +53,9 @@ export function warnOfPropErrors(props) {
     );
   });
 }
+
+export function isElementInsidePaprikaPopover(element) {
```

Comment:
Not sure we should be relating the Panel Component to the Popover in this way. Wonder if this kind of thing can be agnostic of the component type/name

Reply:
Perhaps could use something like the function I used in https://github.com/acl-services/paprika/pull/984/files#diff-881a787b5655895f8685a371e2d800027c5b395cad9574a2faf72dea1e418287R30 if we really need to know if it is a descendant of that element